reference/sliderjoint.md

```diff
@@ -4,8 +4,8 @@ Derived from [Joint](joint.md).
 
 ```
 SliderJoint {
-  MFNode  device   [ ] # linear motor or linear position sensor
-  SFFloat position 0   # initial position (m)
+  MFNode  device   [ ]   # {Linear Motor, Linear PositionSensor, PROTO}
```

Comment:
* To be consistent with the other descriptions (see Track/HingeJoint node for example):
  * "Linear Motor" -> "LinearMotor"
  * "Linear PositionSensor" -> "PositionSensor"


* Missing Brake